deps.edn

```diff
@@ -10,7 +10,8 @@
          com.andrewmcveigh/cljs-time {:mvn/version "0.5.2"}
          com.taoensso/timbre         {:mvn/version "4.10.0"}
          hickory                     {:mvn/version "0.7.1"}
-         com.cognitect/transit-cljs  {:mvn/version "0.8.248"}}
+         com.cognitect/transit-cljs  {:mvn/version "0.8.248"}
+         mvxcvi/multihash            {:mvn/version "2.0.3"}}
```

Comment:
why do we need a new dependency?

Reply:
to work with multihashes

Reply:
So I take it there is no way to inline that functionality? Every dep increases our attack surface